### ISSUE TITLE
sizeBytes should be long (Int64) instead of int (Int32)

### ISSUE THREAD

**Issue by candritzky** [VIP-Logic/NBomber#559]
The data size that can be reported via the Response.Ok(sizeBytes: ...) parameter is currently an int (32-bit).
This leads to an overflow if larger data transfers (in the GB range) are measured.

This leads to a (noticable) error/exception when reporting a single large data transfer.
It also leads to more subtle ("unnoticable") errors when calculating the total sum of the aggregated report.
There seem to be (silent) overflows.

Can we make this a long (64-bit)?

**Comment by AntyaDev** [comment]
Hi @candritzky ,
Yes, it's possible, and by the way, it's a very good task to contribute to open source.

Will you be abe to prepare PR? I can assist with it.

**Comment by candritzky** [comment]
Hi @AntyaDev, I did my very best to prepare two PRs:
1. One for NBomber.Contracts to extend the data type from int to int64 for the respective properties.
2. Then another one for NBomber (which depends on the changes in NBomber.Contracts).
 
The PackageReference version to NBomber.Contracts still needs to be adapted in NBomber once a NuGet is available.
I found it pretty hard because of the separation into two repos.

Not sure if this needs to be considered a breaking change and increment the major version to 5.
